### ISSUE TITLE
Segfault with r2 /bin/ls -c 's 0; pz'

### ISSUE THREAD

**Issue by fcasal** [radare/radare2#9430]
### Work environment

| Questions                                            | Answers
|------------------------------------------------------|--------------------
| OS/arch/bits                             | 16.04.1 Ubuntu x86_64
| File format of the file you reverse      | ELF
| Architecture/bits of the file            | x86/64
| r2 -v          |radare2 2.4.0-git 17371 @ linux-x86-64 git.2.2.0-502-g9c12c4f commit: 9c12c4fe2021fb035797e474c94a9a05c62522c7 build: 2018-02-19__17:55:25
| r2 -V | https://gist.github.com/fcasal/75b79f13ad691d3145cdd7b8b1c4b73a

### TLDR
Segfault caused by null deref in print zoomed.

### Steps to reproduce the behaviour
```
mandlebro@machine:$ r2 /bin/ls -c 's 0;pz'
Segmentation fault (core dumped)
```

The offending line is 5678 of cmd_print.c
```c
5677				RList *list = r_core_get_boundaries_prot (core, -1, NULL, "zoom");
5678				RListIter *iter1 = list->head;
```
where list is NULL.

**Comment by sivaramaaa** [comment]
seeking to offset 0x0 and pz , should return a valid list , i could'nt repro the same here 
But still there are cases where the list could be NULL , like setting zoom.from , zoom.to  to invalid address  etc ..
Maybe u have set something in .radre2rc 
So fixing it , thanks for reporting
